mirgecom/profiling.py

```diff
@@ -94,9 +181,54 @@ def _finish_profile_events(self) -> None:
 
         self.profile_events = []
 
-    def tabulate_profiling_data(self) -> pytools.Table:
+    def get_profiling_data_for_kernel(self, kernel_name: str,
+                                      wait_for_events=True) -> float:
+        """Return value of profiling result for kernel `kernel_name`."""
+        if wait_for_events:
+            self._finish_profile_events()
+
+        def _gather_data(results_list: list):
+            results = [key for key in results_list if key.name == kernel_name]
+            num_calls = 0
+            times = []
+            flops = []
+            bytes_accessed = []
+            # fprint_bytes = []
+
+            for key in results:
+                value = results_list[key]
+
+                num_calls += len(value)
+
+                times += [v.time / 1e9 for v in value]
+                flops += [v.flops / 1e9 if v.flops is not None else 0
+                         for v in value]
+
+                bytes_accessed += [v.bytes_accessed / 1e9
+                              if v.bytes_accessed is not None else 0 for v in value]
+                # fprint_bytes += [v.footprint_bytes if v.footprint_bytes
+                #                  is not None else 0 for v in value]
+
+                del results_list[key]
+
+            return num_calls, times, flops, bytes_accessed
+
+        num_calls, times, flops, bytes_accessed = \
+            _gather_data(self.profile_results)
+
+        if num_calls == 0:
```

Comment:
This has a high risk for being incorrect. What if an elwise kernel and a loopy kernel happen to be called the same thing?